buildSrc/src/main/java/org/elasticsearch/gradle/FileContentsTask.java

```diff
@@ -57,8 +59,12 @@ public void setFile(File file) {
         this.file = file;
     }
 
+    public void setFile(String file) {
+        this.file = new File(file);
```

Comment:
Please use `getProject().file()` here to be consistent with how Gradle resolves strings to files. 
For relative paths, `new File()`  is relative to the current working directory, while `project.file()` is relative to `project.projectDir`.